conf/dev.yaml

```diff
@@ -1,5 +1,10 @@
 plugins:
   hapi-logr:
+    _priority: -5
```

Comment:
don't need priority twice.  also need to set flat to null so you don't get double reporters